src/bin/tikv-server.rs

```diff
@@ -96,14 +113,23 @@ fn build_raftkv(matches: &Matches,
     (create_raft_storage(node, engine).unwrap(), raft_router)
 }
 
-fn get_store_path(matches: &Matches) -> String {
-    let path = matches.opt_str("s");
-    if path.is_none() {
-        return TEMP_DIR.to_owned();
-    }
+fn get_store_path(matches: &Matches, config: &toml::Value) -> String {
+    let path = match matches.opt_str("s") {
```

Comment:
Use if let pattern to make code clean.
if let Some(path) = matches.opt_str("s") {
        return path;
    }

    let path = match config.lookup("server.store")